### PR TITLE
chore: make retry attributes internal

### PR DESCRIPTION
**Pull request by mxschmitt**
https://github.com/microsoft/playwright-dotnet/issues/2316

### CODE REVIEW

File: src/Playwright.MSTest/Services/BrowserService.cs [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 namespace Microsoft.Playwright.MSTest.Services;
 
-public class BrowserService : IWorkerService
+internal class BrowserService : IWorkerService
```
Comment (mxschmitt) [inline src/Playwright.MSTest/Services/BrowserService.cs:30]: drive-by: should not be public.

File: src/Playwright.NUnit/BrowserService.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@
 
 namespace Microsoft.Playwright.NUnit;
 
-public class BrowserService : IWorkerService
+internal class BrowserService : IWorkerService
```
Comment (mxschmitt) [inline src/Playwright.NUnit/BrowserService.cs:30]: drive-by: should not be public.

**Review by pavelfeldman — APPROVED** [review]

**Review by dgozman — APPROVED** [review]
